Gemfile.lock

```diff
@@ -1,6 +1,6 @@
 GIT
   remote: https://github.com/DEFRA/waste-carriers-engine
-  revision: 66e06b533ec6c015c58a50779385b903995db40b
+  revision: de2c090f1a12fb3fc59aebe1e2e68606743ccb04
```

Comment:
Shall we leave this to dependabot?

Reply:
I think I go the other way. If a change in a host app requires an updated version of the engine, I'd rather the update is done with the change, rather than separately. This way its clearer to me that change depended on that version of the engine.